src/Util/Security.php

```diff
@@ -120,9 +120,21 @@ public static function getPermissionsSQLByUserId($owner_id, $remote_verified = f
 			 */
 
 			if (!$remote_verified) {
-				if (DBA::exists('contact', ['id' => $remote_user, 'uid' => $owner_id, 'blocked' => false])) {
+				$cid = 0;
+
+				if (!empty($_SESSION['remote'])) {
+					foreach ($_SESSION['remote'] as $visitor) {
```

Comment:
You can replace this line and the line before by this:
```php
foreach (\Friendica\Core\Session::get('remote', []) as $visitor) {
```

Reply:
No problem, I just pushed a commit to use the Session::get() function instead.